en/docs/learn/kubernetes-operators/k8s-api-operator/k8s-api-operator.md

```diff
@@ -6,11 +6,11 @@ As microservices are increasingly being deployed on Kubernetes, the need to expo
 
 ## Quick Start Guide
 
-[Quick Start Guide v1.1.0](https://github.com/wso2/k8s-api-operator/blob/v1.1.0/README.md).
+[Quick Start Guide v1.2.0](https://github.com/wso2/k8s-api-operator/blob/v1.2.0/README.md).
```

Comment:
Version should be 1.2.2 right?

Reply:
Fixed with 7d7c2002d5a5014897eaec12900bdb6447501d28